test/unit/pages/patientdata.test.js

```diff
@@ -3814,6 +4012,22 @@ describe('PatientData', function () {
       it('should map working.fetchingAssociatedAccounts to fetchingAssociatedAccounts', () => {
         expect(result.fetchingAssociatedAccounts).to.equal(state.working.fetchingAssociatedAccounts);
       });
+
+      it('should map working.addingData to addingData', () => {
+        expect(result.addingData).to.equal(state.working.addingData);
+      });
+
+      it('should map working.updatingDatum to updatingDatum', () => {
+        expect(result.updatingDatum).to.equal(state.working.updatingDatum);
+      });
+
+      it('should map working.queryingData to queryingData', () => {
+        expect(result.queryingData).to.equal(state.working.queryingData);
+      });
+
+      it('should map blip.data to data', () => {
```

Comment:
`blip.data` -> `working.data`? (and another instance lower)

Reply:
Yep.  Fixed both occurrences